.github/actions/setup/action.yml

```diff
@@ -39,11 +39,15 @@ runs:
           !**/node_modules
         key: ${{ github.sha }}
     - uses: actions/cache@v2
+      id: npm-cache
       with:
-        path: |
-          node_modules
-          **/node_modules
-        key: ${{ hashFiles('**/package-lock.json') }}
+        path: ~/.npm
```

Comment:
> Do not cache node_modules across builds. This bypasses more intelligent caching packaged with npm or yarn, and can cause issues with Cypress not downloading the Cypress binary on npm install.

https://docs.cypress.io/guides/continuous-integration/introduction#Caching

Reply:
Looks good, this seems to be the same directory that [`actions/setup-node` uses](https://github.com/actions/setup-node/blob/main/src/cache-utils.ts) when `cache: true` is set on it.

I don't remember the exact reason why I didn't use `setup-node` to cache dependencies.

Reply:
Does that mean we should use this action?